e2e/helpers/browser_helper.js

```diff
@@ -100,6 +104,28 @@ module.exports = class BrowserHelpers extends Helper {
     }
   }
 
+  async grabAttribute(locator, attr) {
+    const elements = await this.locateSelector(locator);
+
+    let getAttribute;
+
+    if(this.isPuppeteer()){
+      getAttribute = async (element, attr) =>  (await element.getProperty(attr)).jsonValue();
+    } else {
+      getAttribute = async (element, attr) => await element.getAttribute(attr);
+    }
+
+    const texts = elements.map(async element => getAttribute(element, attr));
+
+    if (texts.length > 1) {
+      throw new Error(`More than one element found for locator ${locator}`);
+    } else if (texts.length === 1) {
+      return texts[0];
+    } else {
+      return undefined;
```

Comment:
I think that this can be reduced to just `return texts[0]`, if the length is 0 then the result is undefined

Reply:
done